scale/storage/serializers.py

```diff
@@ -117,4 +117,11 @@ class ScaleFileBaseSerializer(ModelIdSerializer):
 
 class ScaleFileSerializer(ScaleFileBaseSerializer):
     """Converts Scale file model fields to REST output"""
-    workspace = WorkspaceSerializer()
+
+    file_type = serializers.CharField()
+    data_type = serializers.CharField()
+    file_path = serializers.CharField()
+
+    source_started = serializers.DateTimeField()
+    source_ended = serializers.DateTimeField()
```

Comment:
Does anything use the base serializer? Otherwise the optional geospatial fields should probably be here instead of the base serializer.